python/GafferImageUI/ImageViewUI.py

```diff
@@ -1167,6 +1180,43 @@ def _updateFromPlug( self ) :
 		for i in self.__row[1:]:
 			i.setEnabled( m != "" )
 
+	def __dragEnter( self, tabbedContainer, event ) :
+
+		result = False
+		if isinstance( event.data, Gaffer.Node ) :
+			result = True
+		elif isinstance( event.data, Gaffer.Set ) :
+			if event.data.size() and isinstance( event.data[0], Gaffer.Node ) :
+				result = True
+
+		if result :
+			for i in self.__row:
+				i.setHighlighted( True )
```

Comment:
I do like the bigger drop target provided by `_CompareParentPlugValueWidget`, but I think it might be clearer if we just highlighted the "pinning" widget. The same way a whole StandardNodeGadget can be the drop target for a connection, but we highlight the specific input plug that will be affected.

Reply:
Addressed.